lib/style.css

```diff
@@ -69,11 +79,15 @@ li div.newcomment button {
 }
 
 #new textarea {
-    max-width: 79em;
+    max-width: 1000px;
```

Comment:
I don't have a good reason for this one. I'm not sure why changed it, I'll change it back, it seems to work on mobile either (it should anyway.)